client/src/components/AboutPage/AboutHeroSection/AboutHeroSection.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FunctionComponent, MutableRefObject } from 'react';
+import { Link } from 'react-router-dom';
+
+// Components
+import Button from 'components/Button/Button';
+
+// Assets
+import { Wave } from 'assets/images/svg';
+import heroImg1 from 'assets/images/dashboard-landing.png';
+import heroImg2 from 'assets/images/search-landing.png';
+
+// Utils
+import { handleScrollTo } from 'utils/helpers';
+
+// Styles
+import { Container, Flex } from 'styles/global';
+import * as S from '../AboutPage.styles';
+
+interface AboutHeroSectionProps {
+  scrollToRef: MutableRefObject<HTMLElement | null>;
+}
+
+const AboutHeroSection: FunctionComponent<AboutHeroSectionProps> = ({ scrollToRef }) => (
+  <S.AboutHeroSection>
+    <Wave reversed={false} />
+    <Container>
+      <Flex>
+        <S.AboutHeroContent>
+          <S.AboutHeadline>
+            <span>Weather App</span>
+            Find your weather predictions in your city
+          </S.AboutHeadline>
+          <p>Easy steps to predict the weather and make your day easier</p>
+          <Link to="/">
+            <Button>Get Started</Button>
+          </Link>
+          <Link to="#how-it-works" onClick={() => handleScrollTo(scrollToRef)}>
```

Comment:
have you tried css first? 
https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior

Reply:
I did not. I am not sure where to put it, though. In a `body`, an `AboutHowSection` styles? Tried both, but it does not work. Do I have to do something else? Also, I am not sure if this should work at all. I tried looking into how to scroll or jump to a particular section in a page with an `id` by clicking on `Link`, but haven't found anything. The only solution I managed to found was to use `element.scrollIntoView()`.

Reply:
i've took a look at your code to check smooth-scrolling, so can give you few hints:
1. please use plain `a` tag for anchor links, no `Link` component is required, because it's not a react-router navigating
2. take a look on example in docs, it uses not only `scroll-behavior` (it should be added for container you want to be scrolled), but also defines `height` for container (100% value won't work, but 100vh will - maybe you can explain me why?) and uses `overflow` property

also few minor comments and questions:
1. please fix active state for sidebar
<img width="266" alt="image" src="https://user-images.githubusercontent.com/43438479/192973771-ad7610f2-fb44-4e1d-ac52-1017ddc773c6.png">
2. why have you moved `Header` out of `Layout` component?

Reply:
I have avoided creating yet another `div` for `Layout` just to set this smooth-scroll to work. What I have done is placed the required css in `<GlobalStyle/> ==> body ==>`
```
 #app {
      scroll-behavior: smooth;
      height: 100vh;
      overflow-x: hidden;
    }
```
Also I've tried looking into why this works only with `vh` units and found that "vh units will be sized to the "largest possible viewport". This means 100vh will be larger than the visible height when the URL bar is shown. While if its height is 100% it will always fill exactly the visible height, whether or not the URL bar is shown." (https://developer.chrome.com/blog/url-bar-resizing/).

Also I don't understand how you got sidebar to show both `li` as active. For me it is working fine. Though, I seem to forgot to set `li` named Dashboard as active on `/locations/:locationId` route.

Reply:
"Also I don't understand how you got sidebar to show both li as active. For me it is working fine. Though, I seem to forgot to set li named Dashboard as active on /locations/:locationId route."
i've just navigated to about page :)

Reply:
I still don't understand. It is working for me correctly.

Reply:
dashboard navlink has '/' url, about - '/about'. looks like you need additional check for dashboard navlink active state
<img width="1708" alt="image" src="https://user-images.githubusercontent.com/43438479/193036633-37b33ea2-53e7-4b20-a6dd-04c917edf343.png">

Reply:
Strange. But why I don't get this result?

When I console.log `isActive` for links
```
  { to: '/', name: 'Dashboard' },
  { to: '/about', name: 'About' },
  { to: '/feedback', name: 'Feeback' }
```
I get 
![Screenshot 2022-09-29 160448](https://user-images.githubusercontent.com/111852745/193039180-4f8ba1e3-b4aa-4eb8-a7ac-01bd5d16af2e.png)
![Screenshot 2022-09-29 160647](https://user-images.githubusercontent.com/111852745/193039525-450c4ec7-d6d7-4453-a00a-7a4d7927c599.png)

Reply:
skip it for now